identity/verifier.go

```diff
@@ -2,26 +2,52 @@ package identity
 
 import (
 	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/syndtr/goleveldb/leveldb/errors"
```

Comment:
Are you sure you want to use this package and not `"errors"`?

Reply:
Thanks, fixed